fabric-networking-v0/src/main/java/net/fabricmc/fabric/impl/networking/ServerSidePacketRegistryImpl.java

```diff
@@ -53,36 +53,29 @@ public void addNetworkHandler(ServerPlayNetworkHandler handler) {
 	}
 
 	protected void forEachHandler(Consumer<ServerPlayNetworkHandler> consumer) {
-		Iterator<WeakReference<ServerPlayNetworkHandler>> it = handlers.iterator();
-
-		while (it.hasNext()) {
-			ServerPlayNetworkHandler server = it.next().get();
+		for (Iterator<WeakReference<ServerPlayNetworkHandler>> iter = handlers.iterator(); iter.hasNext();) {
```

Comment:
who wrote this code... no need to put these in weak refereences at all... horrible.

Reply:
Git blame it lol, This just seems to be adapted old code into a for loop